weldx/transformations.py

```diff
@@ -423,6 +438,7 @@ def __init__(
         orientation.name = "orientation"
 
         self._dataset = xr.merge([coordinates, orientation], join="exact")
+        self._dataset.attrs["time_ref"] = time_ref
```

Comment:
We could also move this directly to the `time` coordinate for a more general way of handling time-like coordiantes in xarray (datetime-like or timedelta-like with reference time)
```python
self._dataset.time.attrs["time_ref"] = time_ref
```
What do you think @vhirtham ?

Maybe instead of setting `None` just skip and omit key?

Reply:
Good idea --- done --- should be mentioned in the developer's documentation if standardize this